apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -102,4 +115,24 @@ export class WorkspaceController {
 
     return target;
   }
+
+  private ngNewArgsFromDto(dto: WorkspaceCreateDto): string[] {
+    const { name, options } = dto;
+    const args: string[] = [name];
+
+    const ngNewOptions = options.map((option) => {
+      // If the option value is undefined, assume it's a boolean flag and include only the option name
+      const optionValue = option.value !== undefined ? ` ${option.value}` : '';
+
+      // If the option is interactive, set it to false
+      if (option.name.includes('--interactive')) {
```

Comment:
why "includes"? should be === no?

Reply:
why this logic in the first place?

Reply:
Yeah my bad it should be ===, I'll change soon.

Reply:
> why "includes"? should be === no?

Didn't want to start normalizing the strings but on second thought === makes more sense I'll switch it. 


> why this logic in the first place?

REST apis can't be interactive

Reply:
so let's not include this option in the client no? we can decide what options can be passed and what not

Reply:
We won't include in the client, but I think it makes sense to block it in the API regardless.